torch/csrc/jit/test_jit.cpp

```diff
@@ -870,6 +873,8 @@ void testControlFlow() {
   script::defineMethodsInModule(cu, cf_examples, torch::jit::script::Resolver(), nullptr);
   auto run = [&](const std::string & name, std::vector<at::Tensor> stack) {
     auto graph = cu.get_method(name).graph();
+    // TODO: why isn't this (a required pass) called?
+    EraseNumberTypes(graph);
```

Comment:
We should decide _when_ EraseNumberTypes happens. If it is just so the interpreter doesn't have to handle it then the `Code` constructor should be the _only_ one to call it, and the calls in the graph executor should be removed.